sdk/src/main/java/io/dapr/client/DaprClientGrpc.java

```diff
@@ -253,6 +256,76 @@
     }
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public <T> Mono<Response<List<State<T>>>> getStates(GetStatesRequest request, TypeRef<T> type) {
+    try {
+      final String stateStoreName = request.getStateStoreName();
+      final List<String> keys = request.getKeys();
+      final int parallelism = request.getParallelism();
+      // TODO(artursouza): Handle consistency param once available in bulk API in GRPC.
```

Comment:
is there an issue open for this?

Reply:
There was, this is a no-issue since it was a doc problem in the HTTP API.

Reply:
Removed comment.